app/tests/src/test_ingest_guru_cards.py

```diff
@@ -47,14 +46,13 @@ def guru_s3_file(mock_s3_bucket_resource, sample_cards):
 
 
 @pytest.mark.parametrize("file_location", ["local", "s3"])
-def test__ingest_cards(db_session, guru_local_file, guru_s3_file, file_location):
+def test__ingest_cards(app_config, db_session, guru_local_file, guru_s3_file, file_location):
```

Comment:
Include the `app_config` fixture to mock the `sentence_transformer` embedding model used by `_ingest_cards()`.